config/environments/production.rb

```diff
@@ -31,21 +37,31 @@
   # config.action_dispatch.x_sendfile_header = "X-Sendfile" # for Apache
   # config.action_dispatch.x_sendfile_header = "X-Accel-Redirect" # for NGINX
 
+  # Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  # config.assume_ssl = true
+
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
You'll need to revert this line. We terminate TLS at the cluster edge, not at the app.

Reply:
See [failing deployment](https://argo.eks.integration.govuk.digital/applications/cluster-services/content-data-api?view=tree&orphaned=false&resource=&node=%2FPod%2Fapps%2Fcontent-data-api-547f597ccd-t4lsf%2F0&tab=events)